_data/sites.json

```
@@ -3520,6 +3520,17 @@
         ]
     },
 
+    {
+        "name": "Cocoleech",
+        "url": "https://members.cocoleech.com/settings",
+        "difficulty": "easy",
+        "notes": "Go to <a href='https://members.cocoleech.com/settings'>your profile</a> and press Delete Account.",
```

```suggestion
        "notes": "Go to [your profile](https://members.cocoleech.com/settings) and press 'Delete Account'.",
```

We have enabled markdown-like syntax to simplify links

---

Uh, am I happy to read that! 😃